### DISCUSSION TITLE
User federation using a external AD or LDAP based database

### DISCUSSION THREAD

**Discussion by 0786ShivaniBansal** [JanssenProject/jans#5961] [Q&A] [answered]
Hii Team We need to work on User federation using an external AD or LDAP-based database 
Do we have any proper documentationn to perform this 
can you suggest some methods to perform this
do we have any documentation AD/LDAP synchronization

Thanks
ShivaniBansal

**Comment by 0786ShivaniBansal** [comment]
Hii Team is there any update regarding this waiting for the reply to proceed further.

Thanks

**Comment by nynymike** [comment, marked as answer]
If you are talking about the ability to sync users and attributes from LDAP, there is a new component called **Jans Link** in 1.0.17:
 https://docs.jans.io/head/admin/link/ 
You must use the nightly build to configure this until we release 17... 

Regarding Inbound SAML (i.e. SSO from ADFS), we have not released an inbound SAML project for Agama, but that's the plan.
